public_html/components/com_jed/helpers/jed.php

```diff
@@ -75,20 +36,18 @@ static public function formatImage($filename, $size = 'small'): string
 		}
 
 		// Filename for small image
-		if ($size == 'small')
+		if ($size === 'small')
 		{
 			$imageSize = str_replace('.', '_resizeDown400px175px16.', $filename);
 		}
 
 		// Filename for large image
-		if ($size == 'large')
+		if ($size === 'large')
 		{
 			$imageSize = str_replace('.', '_resizeDown1200px525px16.', $filename);
 		}
 
 		// Use CDN url
-		$imageUrl = 'https://extensionscdn.joomla.org/cache/fab_image/' . $imageSize;
-
-		return $imageUrl;
+		return 'https://extensionscdn.joomla.org/cache/fab_image/' . $imageSize;
```

Comment:
Just a simple person note 😄 
A fixed path to a fabrik cache folder, while fabrik isn't used anymore in the new JED.
A fixed CDN name, while this is managed by the regularlabs plugin doesn't feel logical to me.

Reply:
In my code I am only using relative paths from the webroot.

Reply:
@conconnl @roland-d the whole image setup needs to be reviewed anyway, for all screens/locations. Posted a suggestion for that in Glip as well. But that's something for a separate issue/PR

Reply:
Agree